pkg/transform/gorm/models/receipt.go

```diff
@@ -19,8 +19,9 @@ type MessageReceipt struct {
 }
 
 type Receipt struct {
-	Index    int64 `gorm:"primaryKey"`
-	ExitCode int64
-	GasUsed  int64
-	Return   []byte
+	Index      int64 `gorm:"primaryKey"`
+	ExitCode   int64
+	GasUsed    int64
+	Return     []byte
+	EventsRoot string
```

Comment:
nit: use `types.DbCid` here as done in the `Message` structure.

Reply:
thanks!